release/test_release_gold.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+source ../_liferay_common.sh
```

Comment:
@kiwm, can you please adjust the commit message to be `Create A release ...` instead of `Create AN`?